src/components/Actions/Actions.vue

```diff
@@ -96,7 +96,7 @@ https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-action
 	</div>
 </template>
 <script>
-import { directive as ClickOutside } from 'v-click-outside'
```

Comment:
This does not seem to be needed to fix the issue. If it is just a matter of style because directives should be _lowerCamelCase_ instead of _UpperCamelCase_ then it should be applied too to [the other files in which `v-click-outside` is used](https://github.com/nextcloud/nextcloud-vue/pull/675/files) (preferably all of them in a separate commit ;-) ).

Reply:
It is definitely a matter of style ;)

Reply:
This is indeed definitely a matter of style ;)